docs/docs/troubleshooting/SUPPORT_BOUNDARIES.md

```diff
@@ -0,0 +1,53 @@
+# Support boundaries
+
+This page describes what is within our scope of support for Kubernetes deployments, and what isn't. 
+
+## Supported components
+### Helm charts
+
+Helm is a Kubernetes package manager. It allows us to provide generic YAML templates that you configure with the specific values for your environments.
+
+You are responsible for creating the components that are required by the product for your type of deployment, as describes in the [Prerequisites](../userguide/PREREQUISITES.md). You need to insert the values (hostnames, configuration values) into your specific `values.yaml` file that is used for installation. We provide documentation for different configuration options in the [Configuration guide](../userguide/CONFIGURATION.md) and the [Examples](../examples/EXAMPLES.md).
+
+If you have followed our documentation on how to configure the Helm charts, and you're using correctly created components, we will provide support if you encounter an error in installation. 
+
+If you find any issue, [raise a ticket](https://github.com/atlassian-labs/data-center-helm-charts/issues/new){.external}. If you have general feedback or questions regarding the charts, use [Atlassian Community Kubernetes space](https://community.atlassian.com/t5/Atlassian-Data-Center-on/gh-p/DC_Kubernetes){.external}.
+
+## Unsupported components
+There is a set of required components that you can create in multiple ways. You are responsible for creating them correctly so you can use them in the Helm charts. 
+
+### Kubernetes cluster
+Helm charts need to be installed in the existing Kubernetes cluster. You need to make sure that you have enough privileges to run the application and create all the necessary entities that the Helm charts require. There are also different Kubernetes flavours that might require specific knowledge of how to install the products in them. For example, OpenShift and Rancher have more strict rules regarding container permissions.
```

Comment:
Just wondering if the first sentence is required? i.e. `Helm charts need to be installed in the existing Kubernetes cluster.`